code/modules/mapping/preloader.dm

```diff
@@ -25,7 +25,7 @@ GLOBAL_LIST_INIT(_preloader_path, null)
 		if(what.vars[attribute] == value)
 			var/message = "<font color=green>[what.type]</font> at [AREACOORD(what)] - <b>VAR:</b> <font color=red>[attribute] = [isnull(value) ? "null" : (isnum(value) ? value : "\"[value]\"")]</font>"
 			log_mapping("DIRTY VAR: [message]")
-			GLOB.dirty_vars += message
+			//GLOB.dirty_vars += message // this causes an 'undefined var' error when defined.
```

Comment:
What is this commented out code still doing here?

Reply:
Well it causes a compiler fail when TESTING is on so I decided to keep it in